src/core/etl/src/Flow/ETL/Transformer/StyleConverter/Style.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Transformer\StyleConverter;
```

Comment:
I would change that namespace to `Flow\ETL\Transformer\Rename` as this is not converting style only anymore